app/dashboard/views/alias_contact_manager.py

```diff
@@ -16,9 +16,27 @@
     generate_reply_email,
     parse_full_address,
 )
-from app.errors import CannotCreateContactForReverseAlias
+from app.errors import (
+    CannotCreateContactForReverseAlias,
+    ErrContactErrorUpgradeNeeded,
+    ErrAddressInvalid,
+    ErrContactAlreadyExists,
+)
 from app.log import LOG
-from app.models import Alias, Contact, EmailLog
+from app.models import Alias, Contact, EmailLog, User
+
+
+# TODO: Move this to a scoped config once the global config gets scoped.
```

Comment:
I wonder if we can change the config in the test like that: https://github.com/simple-login/app/blob/ca2b177e02a717d170afdf0920d9fbc9a8b96d87/tests/auth/test_register.py#L20 ?